### DISCUSSION TITLE
Upgrading to 2.5.2 brings back the APP_DEBUG=false issue from 2.5.0

### DISCUSSION THREAD

**Discussion by relaypilot** [livewire/livewire#3256] [7. Bugs (archived)] [answered]
Steps to reproduce:

        "laravel/framework": "^8.40",
        "laravel/jetstream": "^2.3",
        "livewire/livewire": "^2.5",

On local, `APP_DEBUG=false`, the components are not loading (no errors in the console)

Changing to `APP_DEBUG=true` and the components are loading properly.

Downgrading to `2.5.1` solves the issue.

**Comment by sscotti** [comment]
Interesting.  I think I have the same issue.  Definitely have the same issue.  Probably would not have stumbled onto your post if I had not set APP_DEBUG=true and noticed that everything was working again.  The main thing that was broken were the drop-down menus in my header, e.g. <div @click="open = ! open">.

Noticed that they also added something to handle nonces for CSS in:  protected function cssAssets() in LiveWireManager.php for version 2.5.2.  I was sorting of waiting for that also.

**Reply by sscotti** [reply]
That was mentioned here:  https://github.com/livewire/livewire/issues/650

**Comment by joshhanley** [comment]
@relaypilot for some reason the fix applied in 2.5.1 got reverted by mistake. I have submitted a PR with the fix https://github.com/livewire/livewire/pull/3263

Hope this helps!

**Comment by joshhanley** [comment, marked as answer]
@mkwsra a fix has been published. Please upgrade to 2.5.3 🙂

**Reply by sscotti** [reply]
OK, I'll give it try later.  I take it that also has the fix for CSP for Styles as well.

**Reply by sscotti** [reply]
Seems to be working.  Why did the DEBUG setting have something to do with breaking it ?

**Reply by joshhanley** [reply]
I can't comment on the CSP stuff, as I wasn't across that. I think it was merged, but I've never had to worry about CSP so don't know if it works or not. 

But in relation to debug, when you add the `livewireScripts` into your layout, it injects javascript to boot Livewire, which if `APP_DEBUG` is false, is minified. There was an issue with semicolons missing, so when it was minified, it couldn't process the string of commands properly throwing the error. I have added a test that should catch that in the future, as this has happened before.

**Reply by sscotti** [reply]
Nice to know that it is related to minimizing.  Visited your website, check your e-mail.
